components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -279,6 +284,7 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
   public saveTeam(): void {
     this.saveSuccessful = false;
     this.saving = true;
+    this.updateNameForm.controls['name'].disable();
```

Comment:
Couldn't you add `[disabled]="saving"` in the template so the disable/enable will be done automatically? (Notice that you are setting `this.saving` exactly where you are calling `disable()` and `enable()` in this function.)

Reply:
Nope, if you try to use `[disabled]` with an input that is using formbuilder the compiler yells at you to not do that.

Reply:
Really wish you could. I wanted to do it the way you described. Maybe we can. Need to look deeper but it appears formbuilder gets mad.